src/plugins/multiline/index.ts

```diff
@@ -160,6 +176,53 @@ export default class Multiline extends PluginController<Config> {
       const remove = hookIntoOverrideKeystroke(
         Calc.focusedMathQuill.mq,
         (key, _) => {
+          const mq = Calc.focusedMathQuill?.mq;
+
+          if (key === "Shift-Enter" && this.settings.spacesToNewlines) {
+            if (mq) {
+              mq.typedText("   ");
+              this.enqueueVerticalifyOperation(
+                mq.__controller.container.querySelector(".dcg-mq-root-block")!
+              );
+              setTimeout(() => {
+                this.dequeueAllMultilinifications();
+              });
+            }
+
+            return false;
+          }
+
+          if (mq && key === "Backspace" && this.settings.spacesToNewlines) {
+            if (isNextToTripleSpaceLineBreak(mq, -1)) {
+              mq.keystroke("Backspace");
+              mq.keystroke("Backspace");
+              mq.keystroke("Backspace");
+              return false;
+            }
+          }
+
+          // handle arrow nav with spaces2newlines
```

Comment:
We'll have to see if this works well with Better Navigation.

Reply:
Yep